### ISSUE TITLE
When is the next discussion?

### ISSUE THREAD

**Issue by jaekwon** [atomone-hub/genesis#53]
What timezone are you in?
What topic do you want to talk about? 
What app/site/medium works best for you?

**Comment by moul** [comment]
What timezone are you in? CET

What topic do you want to talk about?
- AMA.
- Connect with individuals willing to contribute, determine their areas of expertise, and direct them to relevant issues/areas.
- Clarify the current priorities (requirements for phase 1). For example, I believe we should include PHOTON details in the constitution, but it doesn't have to be finalized at this stage. The goal is to establish boundaries for each topic, so we can determine if a discussion is currently a priority and how much progress we need to make in defining and implementing it for the current milestone.

What app/site/medium works best for you?
- google meet, we can invite people by email with #30

**Comment by FarOutAndCosmic** [comment]
I'm GMT most often. 

Questions on my mind are:

1. I'm keen to clarify what the USP's (unique selling points) of ATOM1 will be & how it differs from ATOM. In layman's terms ideally.
2. What are the best ways to ensure the current atom community does not split, become competitive or feel alienated?
3. Is it relevant that the $ATOM1 Ticker is not different enough from the current ticker? (Comment raised by Sunny.)
4. Governance: I can't articulate well as I am a non tech associate. One thing that's remained with me for months is the abstain vote influencing an outcome even though it's supposedly abstaining. Can governance be overhauled to be a clear improvement over the existing model? That's a win for me..
5. Digressing somewhat. It's possible to have a successful inflation free FIAT, history has proved this by issuing debt free money. But it's predicated upon empowering those of us at the bottom of the pyramid and not those at the top. Is it possible to weight governance to the small holder to avoid these top heavy pyramids that often plague societal development? A cottage industry as opposed to conglomerates and centralised actors gaining traction simply through the shear weight of their collective wealth?
7. Is it fair to describe AtomOne as an insurance against an ATOM failure?
8. Photon.. Am I to understand the dual token model is applied to Atom1?
9. With airdrops, can they be offered to organisations outside of crypto that have bonafide decentralist values?
10. The **centralists** out there inculcate their values on the younger generation via the school syllabus. As decentralists could we apply this same technique via an education program for the younger generations? Sponsored by Atom1 with a view to promote the true decentralists values. Change is about education and educating the younger generation might be the best way to slowly change this status quo.

I'm somewhat naive with things, I'm a simple idealist at heart. My strength, being here, amongst the wise is I am a complete Bitcoiner layman, thus I represent a reasonable yardstick of what the average person out there may be able to comprehend. 

To those that got to this point. Thanks for reading. I'm here to be part of solutions.

J

**Comment by wnmnsr** [comment]
What timezone are you in? 
—Usually AST or EST.

What topic do you want to talk about?
—Defining AtomOne's brand identity system (graphic and conceptual), as well as its selling points and messaging.
—Defining what is to be AtomOne's voice and strategy of communication, especially as the interests of more people start to peak about AtomOne.

What app/site/medium works best for you?
—Google Meet or Zoom

**Comment by ccomben** [comment]
It would be good to hold a fireside later this week if possible. My timezone is Dubai (GST). Perhaps as we progress, we could hold a couple of different calls a week to accommodate different time zones. IMO, there should be different types of calls as well. One call for brainstorming where we take different topics and go into detail, debate, etc. The other should be an AMA/FAQ, which is well-organized, and we keep to a schedule. This would allow people to get the answers they want quickly and we can build up a rich FAQ that grows with each session. The questions with answers that are TBD would pass to the brainstorming call to keep meetings on time and organized.

**Comment by 0xFDg** [comment]
What timezone are you in? 
—GMT

What topic do you want to talk about? 
— Defining how can I help into project based on my skills and my time. 
— How and who will be in charge of foreign relations
— How ATOM1 can help the Cosmos Community and chain
— how to find support from other Atom chains and Teams


What app/site/medium works best for you? 
—Google Meet or Zoom

**Comment by jaekwon** [comment]
In response to community dеmand, we are proposing to schedule the first Townhall for GovNo members TOMORROW on Thursday, Dеcеmbеr 14th, at 9 am PDT / 5 pm UTC. If you'vе cast a NO or NWV votе on [#prop848](https://twitter.com/hashtag/prop848?src=hashtag_click) of thе [@cosmoshub](https://twitter.com/cosmoshub)
 and wish to еngagе in GovNo discussions and contributе, bе surе to join!

**Comment by adrianakalpa** [comment]
Hi all. Following community’s discussions, let’s get the first Townhall for GovNo members on Thursday, Dеcеmbеr 14th, at 9 am PDT / 5 pm UTC. Here is the X Spaces link that was created for the event https://twitter.com/i/spaces/1YqGoDPDzkaJv  
If you have voted NO or NWV on #prop848 of thе Cosmos Hub and wish to еngagе in GovNo discussions and contributе, bе surе to join!

**Comment by YOUAIE** [comment]
> In response to community dеmand, we are proposing to schedule the first Townhall for GovNo members TOMORROW on Thursday, Dеcеmbеr 14th, at 9 am PDT / 5 pm UTC. If you'vе cast a NO or NWV votе on [#prop848](https://twitter.com/hashtag/prop848?src=hashtag_click) of thе [@cosmoshub](https://twitter.com/cosmoshub) and wish to еngagе in GovNo discussions and contributе, bе surе to join!

Hi, I don't think I'll be available tonight but I'd love some clarification?

Jae:

Why did you choose as nickname X: antechristus #343 (you can answer me privately if needed)?
Why do you insist that atom is not a currency? (it is still exchangeable for currency, isn't that a problem? )
I get the impression that you want to create an organization that's sovereign but very likely to lean towards communism or capitalism!

Thanks for your clarification...

**Comment by adrianakalpa** [comment]
Hi all. I have built a tentative agenda for today's  AtomOne GovNo Townhall [here](https://github.com/atomone-hub/genesis/issues/83#issue-2041559041). Please add in your questions or topics that you would like to discuss.
